lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -636,6 +718,26 @@ def post(self, request):
                 self.msg += \
                     u"<font color='red'>{0} {1} = {2} ({3})</font>".format(
                         _('Deleted'), course.location.to_deprecated_string(), course.id.to_deprecated_string(), course.display_name)
+        elif action == 'get_current_tabs':
+            course_id = request.POST.get('course_id', '').strip()
+            self.msg = u"<h4>{title}</h4><br /><p>{message}</p>".format(
+                title=_('Current Tabs:'),
+                message=self.get_current_tabs(course_id),
+            )
+        elif action == 'delete_tab':
+            course_id = request.POST.get('course_id', '').strip()
+            delete_tab_args = request.POST.get('tab_delete', '').strip()
+            self.msg = u"<h4>{title}</h4><br /><p>{message}</p>".format(
+                title=_('Delete Tab Status:'),
+                message=self.delete_tab(course_id, delete_tab_args),
+            )
+        elif action == 'insert_tab':
+            course_id = request.POST.get('course_id', '').strip()
+            insert_tab_args = request.POST.get('tab_insert', '').strip()
+            self.msg = u"<h4>{title}</h4><br /><p>{message}</p>".format(
```

Comment:
We shouldn't need to use a `<BR>` here; if we need additional space between the `H4` and `P`, we should do so via CSS.